Source/WebCore/workers/service/ServiceWorkerRegistration.cpp

```diff
@@ -337,6 +338,13 @@ void ServiceWorkerRegistration::getNotifications(const GetNotificationOptions& f
 
 #endif // ENABLE(NOTIFICATION_EVENT)
 
+CookieStoreManager& ServiceWorkerRegistration::cookies()
```

Comment:
Aren't you missing the corresponding IDL change? Note that the IDL attribute should also be guarded with `[EnabledBySetting=CookieStoreAPIEnabled&CookieStoreAPIServiceWorkerEnabled]`